tools/create-lp-release-branches.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+DRY_RUN_USAGE="usage: env DEVEL_RELEASE=impish UA_VERSION=27.3 SRU_BUG=1942929 LP_USER=username bash tools/create-lp-release-branches.sh"
+DO_IT_USAGE="usage: env DO_IT=1 DEVEL_RELEASE=impish UA_VERSION=27.3 SRU_BUG=1942929 LP_USER=username bash tools/create-lp-release-branches.sh"
+
+if [ -z "$DEVEL_RELEASE" ]; then
+  echo "please set DEVEL_RELEASE"
+  echo "$DRY_RUN_USAGE"
+  exit 1
+fi
+if [ -z "$UA_VERSION" ]; then
+  echo "please set UA_VERSION"
+  echo "$DRY_RUN_USAGE"
+  exit 1
+fi
+if [ -z "$SRU_BUG" ]; then
+  echo "please set SRU_BUG"
+  echo "$DRY_RUN_USAGE"
+  exit 1
+fi
+if [ -z "$LP_USER" ]; then
+  echo "please set LP_USER"
+  echo "$DRY_RUN_USAGE"
+  exit 1
+fi
+
+if [ -z "$DO_IT"  ]; then
+  echo "This is a dry run. To actually run set DO_IT=1"
+  echo "$DO_IT_USAGE"
+else
+  set -x
+  set -e
+fi
+
+for release in xenial bionic focal hirsute
+do
+  echo
+  echo $release
+  if [ -z "$DO_IT" ]; then
+    echo git checkout upload-${UA_VERSION}-${DEVEL_RELEASE} -B upload-${UA_VERSION}-$release
```

Comment:
I think we can store the command in a variable and just reference it on both situations:
```bash
cmd="git checkout upload-${UA_VERSION}-${DEVEL_RELEASE} -B upload-${UA_VERSION}-$release"
if [ -z "$DO_IT" ]; then
  echo $cmd
else:
  $cmd
fi
```

Reply:
Yes good point!